tests/TestUTestMacro.cpp

```diff
@@ -546,8 +546,8 @@ static void _failingTestMethodWithBYTES_EQUAL()
 TEST(UnitTestMacros, FailureWithBYTES_EQUAL)
 {
     fixture.runTestWithMethod(_failingTestMethodWithBYTES_EQUAL);
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 0x61>");
-    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 0x62>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("expected <97 (0x61)>");
+    CHECK_TEST_FAILS_PROPER_WITH_TEXT("but was  <98 (0x62)>");
```

Comment:
@arstrube , Do you mean SIGNED_BYTES tests missed or I need to add brackets for SIGNED_BYTES too?  If I understand correctly, you decide to add brackets hex format for SIGNED_BYTE later?